llmfoundry/data/finetuning/tasks.py

```diff
@@ -851,6 +854,14 @@ def build_from_hf(
         Returns:
             Dataset: The tokenized dataset.
         """
+        if hf_kwargs is None:
+            hf_kwargs = {}
```

Comment:
Default this to empty dict

Reply:
you can't default mutable objects in python arguments because then each call will share the same object

Reply:
did you mean something else?

Reply:
Nope, forgot about this, my bad